### PR TITLE
Bah 19475

### PR DESCRIPTION
**Pull request by NateJohnson-BAH**
## Description
https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/vets.gov-team/19475

As a developer, I need to reorder the questions on the Facility profile page so that the UX team can test the usability in the demo environment.

## Testing done
confirmed in local UI

## Screenshots
Screen Shot 2019-08-14 at 9 24 25 AM

## Acceptance criteria
- [ ] After the "School Calendar" question the order of the questions in the "Calculator Fields" section is the following:
How do you want to take classes?
Where will you take the majority of your classes? (followed by conditional questions as applicable)
Will the majority of your classes be on the main campus?
Eligible for kicker bonus?

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by shawkey-daniel — CHANGES_REQUESTED** [review]
For the online classes question, there is a OnlineClassesFilter.jsx component you can pull in so you don't duplicate the content. Look at the LandingPage.jsx container for example.

**Review by sonntag-adam — APPROVED** [review]

File: src/applications/gi/components/profile/CalculatorForm.jsx [hunk 1]
```diff
@@ -516,6 +553,111 @@ class CalculatorForm extends React.Component {
...
+        </div>
+      );
+    }
+
+    return (
+      <div>
+        <RadioButtons
+          label={
+            <span>
+              {'Where will you take the majority of your classes?'} <br />(
```
Comment (zurbergram) [inline src/applications/gi/components/profile/CalculatorForm.jsx]: Move the parentheses inside of button to be consistent with other places in CT

**Review by zurbergram — CHANGES_REQUESTED** [review]
Please use correct store object for OnlineClassesFilter and use eslint6 lambda to remove need to bind in the constructor

**Review by zurbergram — APPROVED** [review]
LGTM

**Review by shawkey-daniel — APPROVED** [review]
LGTM

**Comment by hughes-dustin** [comment]
@department-of-veterans-affairs/frontend-review-group Could I get a code review on this at your earliest convenience. Thank you!

**Review by rianfowler — APPROVED** [review]

**Pull request merged**